sources/packages/backend/apps/queue-consumers/src/processors/schedulers/cra-integration/cra-process-integration.scheduler.ts

```diff
@@ -28,7 +28,7 @@ export class CRAProcessIntegrationScheduler extends BaseScheduler<void> {
   @Process()
   async processIncomeVerification(
     job: Job<void>,
-  ): Promise<CRAValidationResultQueueOutDTO> {
+  ): Promise<CRAValidationResult> {
```

Comment:
IMO we can use the return type as QueueProcessSummaryResult and push the `generatedFile` and `uploadedResult` as summary items. 
![image](https://user-images.githubusercontent.com/54600590/209887307-21ab4d31-05ef-43ff-8455-4c7a338dde3a.png)

Reply:
https://github.com/bcgov/SIMS/pull/1624#discussion_r1058657027

Reply:
Optional.